src/mechanics/timeIntegration/NewmarkBase.cpp

```diff
@@ -28,35 +28,18 @@
 #include "math/SparseMatrixCSRSymmetric.h"
 #include "mechanics/structures/StructureOutputBlockVector.h"
 
-//! @brief constructor
-//! @param mDimension number of nodes
 NuTo::NewmarkBase::NewmarkBase (StructureBase* rStructure)
     : TimeIntegrationBase (rStructure)
 {
-	mMuDampingMass = 0.;
-	mToleranceForce = 1e-6;
-	mMaxNumIterations = 20;
-	mBeta = 0.25;
-	mGamma = 0.5;
-	mInternalEnergy = 0.;
-	mExternalEnergy = 0.;
-	mKineticEnergy = 0.;
-	mDampedEnergy = 0.;
-
-	mUseLumpedMass = false;
 }
 
-//! @brief merges the dof values depending on the numTimeDerivatives and rMergeAll
```

Comment:
Not all doxygen code. `\todo` certainly makes sense in source files while \brief, \return, \param, etc. do not imo.